packages/entities/entities-routes/README.md

```diff
@@ -43,6 +44,24 @@ import { RouteList, RouteForm, RouteConfigCard } from '@kong-ui-public/entities-
 import '@kong-ui-public/entities-routes/dist/style.css'
 ```
 
+### Reduce the bundle size (Optional)
+
+This package installs `@kong-ui-public/expressions` and `monaco-editor` to support [Kong's Expressions language](https://docs.konghq.com/gateway/latest/reference/expressions-language/)-related features. However, this will add several megabytes to your bundle size.
+
+If you don't need Expressions features, but have imported the components with support for these features (e.g., `routeFlavor` in `<RouteForm.vue />`) in your host app, add the following configuration to your Vite config to avoid bundling them with your build result:
+
+> As of now, only `<RouteForm.vue />` has Expressions features. You should ensure you are not using Expressions features in your host app before adding this configuration.
+
+```ts
+defineConfig({
+  build: {
+    rollupOptions: {
+      external: ['@kong-ui-public/expressions', 'monaco-editor'],
+    },
+  },
+})
```

Comment:
I still don't think this is needed even with the optional dependencies; the host app isn't going to install them by default.

Reply:
You are right. I've updated the documentation.